integration-libs/epd-visualization/_index.scss

```diff
@@ -1,5 +1,4 @@
 @import '~@spartacus/styles/scss/core';
-@import '~@spartacus/styles/scss/app';
```

Comment:
Just to clarify from my previous comment before from the other PR. Are you sure we can just delete this? Was there anything needed from it in the epd-visualization library

(I took a peek, and I don't think so)